README.md

```diff
@@ -86,6 +86,19 @@ Then, use it in your template:
 </div>
 ```
 
+### `modifier` function
+
+The native class example can be implemented using `modifier` function, which calls passed function argument on `didReceiveArguments`.
+
+```js
+// app/modifiers/scroll-top.js
+import { modifier } from 'ember-oo-modifiers';
+
+export default modifier(function scrollTop(element, [scrollPosition]) {
```

Comment:
Can we name what `[scrollPosition]` is since this is documentation destructuring obfuscates what is actually passed here. Good for prod code not great for documentation. Least I was confused what this function would be passed in. Especially since the first argument is `element`.